pkg/docker/docker_util.go

```diff
@@ -192,3 +201,26 @@ func GetDockerClient() (Docker, error) {
 	}
 	return Client, nil
 }
+
+// ExecCommend will execute a command in container and returns an execution id
+func ExecCommend(ctx context.Context, cli Docker, containerID string, command []string) (types.IDResponse, error) {
+	ExecConfig.Cmd = command
+	r, err := cli.ContainerExecCreate(ctx, containerID, ExecConfig)
+	if err != nil {
+		return types.IDResponse{}, err
+	}
+	return r, err
+}
+
+func InspectExecResp(ctx context.Context, cli Docker, containerID string) error {
+	resp, err := cli.ContainerExecAttach(ctx, containerID, types.ExecStartCheck{})
+	if err != nil {
+		return err
+	}
+
+	s := bufio.NewScanner(resp.Reader)
+	for s.Scan() {
+		fmt.Println(s.Text())
```

Comment:
Should we use fmt.Println or logger.Print ?

Reply:
no that will  print the output in json and we need plain output

Reply:
ummm... you can configure the logger to output raw text... it'll have the timestamp but will be in raw text not json.
https://github.com/flyteorg/flytestdlib/blob/master/logger/config.go#L17

Reply:
discussed offline... Yuvraj convinced me to keep it as is